ci_framework/roles/repo_setup/tasks/rhos_release.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Install RHOS Release tool
+  ansible.builtin.package:
+    name: "{{ cifmw_repo_setup_rhos_release_rpm }}"
+    state: present
+
+- name: Print the rhos-release tool version
+  become: true
+  ansible.builtin.command: rhos-release --version
```

Comment:
I'm not sure this will actually be shown, unless you have a debug task for it.. ?